sw/airborne/modules/nav/nav_rotorcraft_hybrid.c

```diff
@@ -83,15 +94,21 @@ float nav_hybrid_pos_gain = 1.0f;
 bool force_forward = 0.0f;
 #endif
 
-#ifndef NAV_HYBRID_GOTO_MODE
-#define NAV_HYBRID_GOTO_MODE NAV_SETPOINT_MODE_SPEED
+/* When using external vision, run the nav functions in position mode for better tracking*/
+#ifndef NAV_HYBRID_EXT_VISION_SETPOINT_MODE
+#define NAV_HYBRID_EXT_VISION_SETPOINT_MODE FALSE 
 #endif
 
+/* Use max target groundspeed and max acceleration to limit circle radius*/
+#ifndef NAV_HYBRID_LIMIT_CIRCLE_RADIUS
+#define NAV_HYBRID_LIMIT_CIRCLE_RADIUS FALSE 
+#endif
 /** Implement basic nav function for the hybrid case
  */
 
 static void nav_hybrid_goto(struct EnuCoor_f *wp)
 {
+  nav_max_acceleration_sp = nav_hybrid_max_acceleration / 2.0;
```

Comment:
Why this magic division by two?

Reply:
I have made it more clear now. I have experienced that some platforms (like my RW3C) benefit from a lower acceleration bound in certain phases but still need the full acceleration limit to perform tight turns. I made a new variable that if not defined defaults to the maximum acceleration.